### PR TITLE
Update pre-cache docs to mention filtering and space checking

### PR DESCRIPTION
**Pull request by sudomakeinstall2**

### CODE REVIEW

File: docs/pre-cache/README.md [hunk 1]
```diff
@@ -24,6 +24,7 @@ Maintenance windows configuration is not provisioned by TALO. A user applies TAL
     - References to the policies containing the required release and operator versions for this set
     - The need for image pre-caching
 - User applies the TALO CR to the hub cluster at the beginning of the maintenance window
```
Comment (jc-rh) [inline docs/pre-cache/README.md:26]:
```suggestion
- User applies the ClusterGroupUpgrade CR to the hub cluster at the beginning of the maintenance window
```

**Review by sakhoury — APPROVED** [review]
/lgtm

**Pull request merged**